sources/gbts/cmd/local/ssh.sh

```diff
@@ -1,9 +1,38 @@
+function remote_ssh_command {
```

Comment:
I would name this function `gbt__is_ssh_command` and moved it into the `sources/gbts/cmd/_common.sh` as we might need it in the remote `gbt_ssh` as well (please test). Also the `gbt_vagrant` could benefit from this (local and remote).

Reply:
> .. as we might need it in the remote gbt_ssh as well (please test)

sure thing, do I understand you correctly that you want me to test the remote gbt_ssh function as well? (need to read up on that first though)

Reply:
Yes, it would be nice if your change would also work via [prompt forwarding](https://github.com/jtyr/gbt#prompt-forwarding).

Reply:
Ok, I'll take a look at that

Reply:
I'll added a check in remote/ssh.sh 1ff3099678969f8ea2ada0d51298c105367493b2

Reply:
Please could you implement it also into the local & remote `gbt_vagrant`?

Reply:
Will do @jtyr 
Will probably take a couple of days though as I'm quite busy.
Thanks for the coaching, I'm learning a lot of git related stuff here 👍

Reply:
I didn't see the same behavior when using 'gbt_vagrant ssh'
This is tested with vagrant 2.2.4
![vagrant](https://user-images.githubusercontent.com/254416/56017959-0ca4fb00-5d01-11e9-82b2-b5cba2033d9d.png)

This is the behavior with the current (gbt 1.2.0) version of gbt_vagrant
![gbt_vagrant](https://user-images.githubusercontent.com/254416/56018000-2b0af680-5d01-11e9-99e6-c8264442a17f.png)

I have however created patch that doesn't build the PS1 when calling vagrant ssh with `-c` or `--command`, I think it's redundant to build the PS1 when it's never displayed.

I've tested both the local and remote gbt_vagrant
![patched_gbt_vagrant](https://user-images.githubusercontent.com/254416/56018091-67d6ed80-5d01-11e9-9bd1-d958bf6aed12.png)

Reply:
Nice work. I would prefer if vagrant and ssh functions would use the same function somehow but I think the current solution is good enough. Please test it again with my modifications and if you see no issue, we are ready to merge it.

Reply:
Btw. you got a really nice shell prompt! ;o)

Reply:
That's your code man, so thanks for the prompt